requirements/nominal_test.txt

```diff
@@ -4,3 +4,4 @@
 pandas >1.0.0, <=2.0.2 # cannot pin version due to numpy version incompatibility
 dython <=0.7.4
 scipy >1.0.0, <1.11.0 # cannot pin version due to some version conflicts with `oldest` CI configuration
+statsmodels >= 0.14.0
```

Comment:
Don't we wanna add any upper bound like it's done for other dependencies?

Reply:
you are right, let me add one :]